src/JustSaying/Messaging/Channels/Receive/MessageReceiveController.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace JustSaying.Messaging.Channels.Receive;
+
+public class MessageReceiveController : IMessageReceiveController, IMessageReceiveThreadPausingController
+{
+    private readonly ManualResetEvent _manualResetEvent = new(false);
+
+    public void Stop()
+    {
+        ShouldStopReceiving = true;
+    }
+
+    public void Start()
+    {
+        _manualResetEvent.Set();
+        _manualResetEvent.Reset();
+        ShouldStopReceiving = false;
+    }
+
+    public void PauseThreads(CancellationToken cancellationToken)
```

Comment:
Considering the interface is internal, should this method be public rather than explicitly implementing the interface?